config/sections/mixins/parent.php

```diff
@@ -22,6 +20,14 @@
                 if (!$parent) {
                     throw new Exception('The parent for the query "' . $query . '" cannot be found in the section "' . $this->name() . '"');
                 }
+
+                if (
+                    is_a($parent, 'Kirby\Cms\Page') === false &&
+                    is_a($parent, 'Kirby\Cms\Site') === false &&
+                    is_a($parent, 'Kirby\Cms\User') === false
```

Comment:
I think we need to include File here as well. You can add a files section to a file blueprint for example and the files section will automatically use the siblings.

Reply:
But would the File be the object in the parent prop then?

Reply:
You are right, it wouldn't. But it could still be useful I guess. Would it do any harm to check for the file here as well?

Reply:
Probably not... then I would check for ModelWithContent?